content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -170,13 +173,22 @@ spec:
 When you create the Ingress with `kubectl create -f`:
 
 ```shell
-$ kubectl get ing
-NAME      RULE          BACKEND   ADDRESS
-test      -
-          foo.bar.com
-          /foo          s1:80
-          /bar          s2:80
+kubectl describe ingress test
+```
+
+```shell
```

Comment:
nit: I'm not sure how much info one expects to see from this snippet, but it might help for completeness to show everything

```
Name:             app-health
Namespace:        kube-system
Address:          169.X.X.X
Default backend:  default-http-backend:80 (<none>)
TLS:
  cmltest terminates alb.example.cloud
Rules:
  Host  Path  Backends
  ----  ----  --------
  *     *     default-http-backend:80 (<none>)
Annotations:
  ingress.example.net/server-snippets:  location = / { return 200 'healthy'; add_header Content-Type text/plain; }
Events:                                 <none>
```

Reply:
Thanks @cmluciano! 🙇

Reply:
I got it and I'll fix soon.
Thank you!